docs/Compile.rst

```diff
@@ -361,9 +361,10 @@ To install Chocolatey and the required dependencies:
 
 * Paste in the Chocolatey install command and hit enter
 * Close this ``cmd.exe`` window and open another Admin ``cmd.exe`` in the same way
-* Run the following command::
+* Run the following commands::
 
-    choco install git cmake strawberryperl -y
+    choco install git cmake.portable strawberryperl -y
+    cpanm XML::LibXML
```

Comment:
Also, the cpan command won't work until a new cmd.exe is opened because PATH won't be updated:

```
C:\Users\Media>choco install strawberryperl -y
Installing the following packages:
strawberryperl
By installing you accept licenses for the packages.

StrawberryPerl v5.20.1.1
 Installing StrawberryPerl...
 StrawberryPerl has been installed.
 The install of strawberryperl was successful.

Chocolatey installed 1/1 package(s). 0 package(s) failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

C:\Users\Media>cpanm XML::LibXML
'cpanm' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Media>
```